recipes/bioconductor-rhdf5lib/meta.yaml

```diff
@@ -21,8 +23,12 @@ build:
 requirements:
   host:
     - r-base
+    - zlib
+    - libgfortran
```

Comment:
is `- {{ compiler('fortran') }}` not working?

Reply:
Didn't know this is equivalent. I thought it just wants to link against fortran, not building fortran code. But I might be wrong.